docs/uperf.md

```diff
@@ -18,19 +18,24 @@ metadata:
   name: uperf-benchmark
   namespace: ripsaw
 spec:
-  name: uperf
-  args:
-    hostnetwork: false
-    pin: true
-    pin_server: "master-0"
-    pin_client: "master-1"
-    # Server size must always be 1 or more
-    pairs: 1
-    proto: tcp
-    test_type: stream
-    nthr: 2
-    size: 16384
-    runtime: 60
+  workload:
+    # cleanup: true
+    name: uperf
+    args:
+      hostnetwork: false
+      pin: false
+      pin_server: "master-0"
+      pin_client: "master-1"
+      rerun: 1
+      pair: 1
+      nthr: 2
```

Comment:
The `nthr` means "number of threads"?  Is this name, `nthr`, what uperf users?  Or just a short hand?

Reply:
Maybe rename it to match the `group.nthreads` in the uperf yml?

Reply:
yup, took `nthr` straight out of uperf.